BaseFilter/sections/specific.txt

```diff
@@ -16802,9 +16802,9 @@ outlook.live.com#?##app > div[class] > div[class] > div[class] > div[class]:not(
 outlook.live.com#?#.customScrollBar[data-is-scrollable] > div[class] > div[class]:not([data-convid]):has(> div:not([id]):not([class]) > div[class] > div[class] > div[class] > div[class] > div[class] > .fbAdLink)
 !
 ! check-host.net
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+|\＋|\﹢|х)$/):upward(2)
-/^https:\/\/check-host\.net\/images\/[a-z]?[0-9]?[0-9]?[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
-check-host.net#?#div:has(> a[href]:contains(/^aeza\.net|100%|Offshore/))
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+|\＋|\﹢|х|Х)$/):upward(2)
+/^https:\/\/check-host\.net\/images\/[a-z]?[a-z]?[0-9]?[0-9]?[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net#%#//scriptlet('remove-node-text', '#text', '/The incredible is real! Profitable VPS 5.7 GHz, 6 locations, and more: |aeza\.net|100% Offshore Server/')
```

Comment:
I don't see such text on my end. It can be removed by this rule.
```adblock
check-host.net#?#div:has(> span > a[href]:contains(100% Offshore))
```

Reply:
Strange it worked fine for me

Reply:
This is what it looks like for me `<a href="/etgo/specialrdp">100% Offshore Server</a>` what does it look like for you

Reply:
Yeah, I have the same. The rule didn't work earlier, but do now.
That's kinda strange, because the tag is not `text`.
Anyway, I'd prefer the ext-css rule as it more accurate and do not interfere with its design.

Reply:
When I first updated the rule, it looked a lot like yours, but I changed it because I figured they would just remove span to bypass it.

Reply:
They could aslo replace text, so like I said before, it's cat&mouse game.

Reply:
I know it was more of a test to see how willing they are to change up the look of their site.